.github/workflows/build_dockers_workflow.yml

```
@@ -13,64 +13,118 @@ name: Build docker image - workflow_call/on-demand
     runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+
+    steps:
+      - name: checkout tari-ootle
+        uses: actions/checkout@v5
+
+      - name: Set Matrix
+        id: set-matrix
+        shell: bash
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2034:warning:2:1: build_items appears unused. Verify use (or export if used externally)

(shellcheck)